backend/business-partner-agent/src/main/java/org/hyperledger/bpa/impl/PartnerManager.java

```diff
@@ -77,10 +80,14 @@ public class PartnerManager {
     @Inject
     BPAMessageSource.DefaultMessageSource ms;
 
-    public List<PartnerAPI> getPartners() {
-        return StreamSupport.stream(repo.findAll().spliterator(), false)
-                .map(converter::toAPIObject)
-                .collect(Collectors.toList());
+    public Page<PartnerAPI> getPartners(@NonNull Pageable pageable) {
+        Page<PartnerAPI> partners = repo.findAll(pageable).map(converter::toAPIObject);
+        return partners;
+    }
+
+    public Page<PartnerAPI> getPartnerByInvitation(
```

Comment:
Naming is confusing. findByStateNotEqualsInvitation() or something similar is more readable.